evalml/objectives/regression_objective.py

```diff
@@ -9,5 +9,5 @@ class RegressionObjective(ObjectiveBase):
     problem_type (ProblemTypes): Type of problem this objective is. Set to ProblemTypes.REGRESSION.
```

Comment:
We should update the docstring for each of the objective classes!

Reply:
Great point!